boards/shields/g1120b0mipi/g1120b0mipi.overlay

```diff
@@ -20,11 +15,17 @@
 		enable-gpios = <&nxp_mipi_connector 32 GPIO_ACTIVE_HIGH>;
 		regulator-boot-on;
 	};
+
+	lvgl_pointer {
+		compatible = "zephyr,lvgl-pointer-input";
+		input = <&ft3267>;
+		invert-y;
+	};
 };
 
 &nxp_mipi_i2c {
 	status = "okay";
-	ft3267@38 {
+	ft3267: ft3267@38 {
```

Comment:
I don't know how these nodes have been spared by the renaming, but they now should be called :
ft3267_<shield_name>.
See https://github.com/zephyrproject-rtos/zephyr/pull/50526.

Not directly related so not blocking.
But would be nice to fix this while you're at

Reply:
Since I touched the files, might as well do this one too :^)